pkg/config/msp_test.go

```
@@ -1314,3 +1258,59 @@ func TestRevokeCertificateFromMSPFailure(t *testing.T) {
 		})
 	}
 }
+
+func baseMSP() MSP {
+	certBlock, _ := pem.Decode([]byte(dummyCert))
+	cert, _ := x509.ParseCertificate(certBlock.Bytes)
+
+	privKeyBlock, _ := pem.Decode([]byte(dummyPrivateKey))
+	privKey, _ := x509.ParsePKCS8PrivateKey(privKeyBlock.Bytes)
```

Why are we ignoring the error?